src/ansys/fluent/core/__init__.py

```diff
@@ -31,7 +31,7 @@
 from ansys.fluent.core.session import BaseSession as Fluent  # noqa: F401
 from ansys.fluent.core.utils import fldoc
 from ansys.fluent.core.utils.fluent_version import FluentVersion  # noqa: F401
-from ansys.fluent.core.utils.search import search  # noqa: F401
+from ansys.fluent.core.utils.search import _search, search  # noqa: F401
```

Comment:
`_search()` can still be tested at `ansys.fluent.core.utils.search`. This import pollutes the main module interface.

Reply:
We can remove the old search function. The new search function should have the `version` argument like the old search to search across the APIs of a given version. I don't think the last argument `search_root` of the old search is very important and it is okay not to support in the new function.

Reply:
@mkundu1 @seanpearsonuk 

Right now files in `api_tree` are based on 242, do we want to keep files for all 222 to 242 ?

We are not generating them each time.